src/audio/codec_adapter/codec_adapter.c

```diff
@@ -246,27 +246,23 @@ ca_copy_from_source_to_module(const struct audio_stream *source, void *buff, uin
 }
 
 static void
-codec_adapter_copy_from_lib_to_sink(const struct module_processing_data *mpd,
```

Comment:
Same comments here. Maybe group the patches together by functionality (look at my previous comment).

Reply:
@dbaluta let me know if this looks better.